open_academy/models/course.py

```diff
@@ -0,0 +1,11 @@
+from odoo import models,fields
+
+
+class course(models.Model):
```

Comment:
Class names follow the upper CamelCase convention the first letter should start with upper letter